.github/workflows/ci.yaml

```diff
@@ -25,18 +25,17 @@ jobs:
   test:
     uses: bazel-contrib/.github/.github/workflows/bazel.yaml@646899086d7aaee8e532540480f3e91e00596234 # 11 Dec 2024
     with:
-      folders: '[".", "e2e/smoke", "e2e/repository-rule-deps", "e2e/system-interpreter", "examples/uv_pip_compile"]'
+      # NB: the root folder is tested with Aspect Workflows on BuildKite, see /.aspect/workflows/config.yaml
+      folders: '["e2e/smoke", "e2e/repository-rule-deps", "e2e/system-interpreter", "examples/uv_pip_compile"]'
       # TODO: Build Windows tools and add to toolchain
       # TODO(alex): switch the root folder to bzlmod
       # TODO: fix remaining folders on Bazel 8
       exclude: |
         [
           {"os": "windows-latest"},
-          {"folder": ".", "bzlmodEnabled": true},
           {"folder": "e2e/repository-rule-deps", "bzlmodEnabled": false},
           {"folder": "e2e/system-interpreter", "bzlmodEnabled": false},
           {"folder": "examples/uv_pip_compile", "bzlmodEnabled": false},
-          {"folder": ".", "bazelversion": "8.0.0"},
```

Comment:
Are you okay with not running the WORKSPACE variant on CI anymore?

Reply:
i believe AW is testing workspace w bazel 7

Reply:
Yup I'm pretty confident we aren't losing any test coverage here.